pkgs/tools/misc/bdf2ttf/default.nix

```
@@ -0,0 +1,29 @@
+{ lib, stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation {
+  pname = "bdf2ttf";
+  version = "unstable-2016-07-03";
+
+  src = fetchFromGitHub {
+    owner = "koron";
+    repo = "bdf2ttf";
+    rev = "1baae7b70a432153e3d876966e47a78f0e572eac";
+    hash = "sha256-235BTcTaC/30yLlgo0OO2cp3YCHWa87GFJGBx5lmz6o=";
+  };
+
+  dontConfigure = true;
+
+  makeFlags = [ "gcc" ];
+
+  installPhase = ''
+    install -Dm755 bdf2ttf $out/bin/bdf2ttf
```

```suggestion
    runHook preInstall

    install -Dm755 bdf2ttf $out/bin/bdf2ttf

    runHook postInstall
```
https://nix.dev/tutorials/packaging-existing-software#phases-and-hooks